src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.html

```diff
@@ -0,0 +1,85 @@
+<p-toast></p-toast>
+<form [formGroup]="addEditUserForm">
+    <div class="card p-fluid">
+        <div class="field">
+            <label for="firstName">First Name * </label>
```

Comment:
This label should come from language constant

Reply:
resolved